docs/creating_new_react_component.md

```diff
@@ -0,0 +1,95 @@
+# Creating a new React Component for v3's website
+
+## Create the component
+
+- Each component should be in its own directory
+- Should be placed in `/app/javascript/components/<componentname>`
+
+## Create the tests
+
+- Place tests in `/test/javascript/components/<component-name>/ComponentName.test.tsx`
+- The tests must use the [JEST](https://jestjs.io/) test framework
```

Comment:
```suggestion
- The tests must use the [Jest](https://jestjs.io/) test framework
```

Reply:
I wasn't sure, and the website has it all in caps. :P thanks!